app/src/mixins/search-mixin.ts

```diff
@@ -35,7 +35,6 @@ export class SearchMixin extends Mixins(CommonMixin) {
   @Getter getLocation!: Location
   @Getter getOriginEntityTypeCd!: EntityTypes
   @Getter getRequestActionCd!: NrRequestActionCodes
-  @Getter getSearchBusiness!: BusinessSearchIF
```

Comment:
GHA warning says that BusinessSearchIF no longer needs to be imported.

Reply:
oops missed removing! Done